omero/sysadmins/version-requirements.rst

```diff
@@ -141,107 +156,116 @@ Components
          - |Broken|
+         - |Broken|
+         - |Recommended|
 
   * Rationale: 2.7 is provided by all systems except for CentOS/RHEL
-    6.x, however it is available officially via SCL. Note that 3.3 is
-    also available via SCL, so there is potential for a 3.x migration
-    in the 5.4 timeframe. OMERO.web **no longer** works with Python 2.6,
-    the OMERO.server can still be used with Python 2.6.
+    6.x, however it is available officially via SCL.
 
 * Ice
 
   .. list-table::
        :header-rows: 1
 
        * - Ice
-         - OMERO 5.3
          - OMERO 5.4
          - OMERO 5.5
+         - OMERO 5.6
        * - 3.5
          - |Deprecated|
-         - |Deprecated|
+         - |Dropped|
          - |Dropped|
        * - 3.6
          - |Recommended|
          - |Recommended|
-         - |Supported|
-       * - 3.7
-         - |Upcoming|
-         - |Upcoming|
          - |Recommended|
+       * - 3.7
+         - |Unsupported|
+         - |Unsupported|
+         - |Unsupported|
+
+  * Rationale: Ice releases often include breaking changes and Ice 3.7
+    requires substantial changes to OMERO. To avoid future maintenance
+    burden OME's effort is better invested in migrating away from Ice
+    completely.
 
 * Java
 
   .. list-table::
        :header-rows: 1
 
        * - Java
-         - OMERO 5.3
          - OMERO 5.4
          - OMERO 5.5
+         - OMERO 5.6
        * - 7
          - |Deprecated|
-         - |Deprecated|
-         - |Deprecated|
+         - |Dropped|
+         - |Dropped|
        * - 8
          - |Recommended|
          - |Recommended|
          - |Supported|
-       * - 9
-         - |Upcoming|
-         - |Upcoming|
+       * - 11
+         - |Unsupported|
+         - |Supported|
          - |Recommended|
 
-  * Rationale: For 5.3 and 5.4 deprecate 7; this is because security support ended in April 2015.
-    Currently, Matlab is shipped with Java 7 so it cannot be marked as dropped yet.
+  * Rationale: For OMERO 5.4 deprecate Java 7 because security support ended
+    in April 2015. MATLAB ships with Java 8 since R2017b.
+
+.. warning::
+
+   *Only* OpenJDK is supported from OMERO 5.5. OME no longer tests Oracle
+   Java due to `licensing changes
+   <https://www.oracle.com/technetwork/java/javase/overview/oracle-jdk-faqs.html>`__.
 
 * nginx
 
   .. list-table::
        :header-rows: 1
 
        * - nginx
-         - OMERO 5.3
          - OMERO 5.4
          - OMERO 5.5
-       * - 1.6
-         - |Deprecated|
-         - |Dropped|
-         - |Dropped|
+         - OMERO 5.6
        * - 1.8
-         - |Supported|
          - |Deprecated|
          - |Dropped|
+         - |Dropped|
        * - 1.10
-         - |Recommended|
          - |Recommended|
          - |Supported|
+         - |Deprecated|
        * - 1.12
+         - |Supported|
+         - |Recommended|
+         - |Supported|
+       * - 1.14
          - |Upcoming|
-         - |Upcoming|
+         - |Supported|
          - |Recommended|
```

Comment:
Nightshade OMERO.web and the next IDR release have the latest Enterprise Linux 7 stable version of Nginx: `nginx-1.16.0-1.el7.ngx.x86_64`, so I think we could say `1.16` is recommended.

Reply:
Though  a quick look at the current distro versions (as opposed to the upstream Nginx repos) shows:
- ubuntu:18.04: 1.14
- centos:7 via EPEL: 1.12
- ubuntu:16.04: 1.10

So it's a toss-up between recommending 1.14 and 1.16

Reply:
This is confusing, there's two Nginx sections in this doc and this one is missing the 1.16 line. Since Nginx isn't as critical as before (it's proxying OMERO.web rather than using FastCGI as in older versions) perhaps this section could be removed to avoid duplication?

Reply:
I don't think I'm the one to judge. I'll add a 1.16 line for now in the hope that larger changes can be in subsequent PRs. :smiley: